### PR TITLE
Exported AvatarProps for the issue #2802

### PR DESCRIPTION
**Pull request by hetanthakkar1**
Exported AvatarProps from index.ts to allow importing AvatarProps from `react-native-elements` instead of `react-native-elements/dist/avatar/Avatar`

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import Image from './image/Image';
 import Card from './card/Card';
 import Tile from './tile/Tile';
 import Avatar from './avatar/Avatar';
+import { AvatarProps } from './avatar/Avatar';
```
Comment (flyingcircle) [inline src/index.ts:25]: This can be merged with the line above:
```js
import Avatar, {AvatarProps} from './avatar/Avatar';
```

**Review by flyingcircle — CHANGES_REQUESTED** [review]
If we're going to export the Prop types, then let's add exports for all of the components. Add exports for all of the components.

**Review by jeeva2812 — COMMENTED** [review]
LGTM. Works for me.

**Comment by hetanthakkar1** [comment]
@flyingcircle  I have made the changes. Could you please review it

**Review by jeeva2812 — APPROVED** [review]

**Comment by hetanthakkar1** [comment]
Okay I am on it. @flyingcircle I have a doubt. I can see d.ts files when I install react -native-elements in a project in node modules.
But I can't see those files in the repo. I want to solve an issue that requires me to change that d.ts files. How can I get that file?

**Pull request merged**